src/pay/payment_producer.py

```diff
@@ -22,6 +22,8 @@
 logger = main_logger
 
 MUTEZ = 1e+6
+BOOTSTRAP_SLEEP = 32
```

Comment:
why 32?

Reply:
semi random pick. Assuming a new node would need several hours, potentially, to bootstrap, I did not want the loop-check happening frequently. 32 blocks is just over 30m; just trying to keep it a power of 2 for no real reason. I'm open to something less if you think we should check more frequently.

Reply:
help me out here: I thought we are coping with the fact that the "bootsrapped" command in tezos-client returns even if it is not bootstrapped yet. For this reason you will call the command again and again through a loop and compare the time message in the return value with the local time of the machine. This is the logic that I would expect from the tezos-client btw...
As long as the bootstrapping is not finished it would be nice to notify the user of the progress that is being made. Therefore the bootstrap loop could call itself every after a new block has been found and a new time value can be compared. I would use "2" in this case?

Reply:
Yes, /bootstraped always returns. Seems to be a stream of data even when using `curl` which I did not realize. I thought that if you used `curl` (aka python requests) that it would fetch 1 entry and exit. So I need to modify the code to handle this.

Yes, this is more/less the logic of the tezos-client but I'm trying to remove usage/dependency of the tezos-client.

I can certainly modify the code to print out more frequently, loop more often.